### PR TITLE
#9 Added Capability to upload Images using URLs.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#9 solved. @deshraj Please review the PR and tell me the necessary changes.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,3 +11,4 @@ pyyaml==3.12
 uwsgi==2.0.13.1
 websocket-client==0.37.0
 daphne==0.15.0
+requests
```
Comment (deshraj) [inline requirements.txt]: @RishabhJain2018 can you please add the version of requests?

Comment (RishabhJain2018) [inline requirements.txt]: yeah sure.

**Comment by RishabhJain2018** [comment]
@deshraj I have updated the PR. Please review it and tell me the necessary modifications.

File: grad_cam/templates/classification/classification.html [hunk 2]
```diff
@@ -246,6 +249,35 @@ <h2 id="userImages">Try Grad-CAM On Your Images</h2>
...
+  var src = document.getElementById("image_url").value;
+  $('#comments').prepend('<br>');
+  $('#comments').prepend('<li>' + "Starting the Image Download process." + '</li>');
+
+  scrollToElement($("#termstart"));
+  scrollToElement($("#resultHeading"));
+
+  $.ajax({
+    type : "POST",
+    url  : "/upload/grad_cam_using_image_url/",
```
Comment (deshraj) [inline grad_cam/templates/classification/classification.html]: Please don't hardcode the URL.

File: grad_cam/templates/vqa/vqa.html [hunk 3]
```diff
@@ -1205,6 +1209,37 @@ <h2 id="userImages">Try Grad-CAM on your images</h2>
...
+  var src = document.getElementById("image_url").value;
+  $('#comments').prepend('<br>');
+  $('#comments').prepend('<li>' + "Starting the Image Download process." + '</li>');
+
+  scrollToElement($("#termstart"));
+  scrollToElement($("#resultHeading"));
+
+  $.ajax({
+    type : "POST",
+    url  : "/upload/grad_cam_using_image_url/",
```
Comment (deshraj) [inline grad_cam/templates/vqa/vqa.html]: Here also.

File: grad_cam/templates/classification/classification.html (hunk 2 above)
Comment (RishabhJain2018) [inline grad_cam/templates/classification/classification.html]: Okk. then should I go with name of the URL. will that be good?

**Comment by RishabhJain2018** [comment]
@deshraj PR updated.

**Pull request merged**